crates/loader/src/assets.rs

```diff
@@ -90,6 +90,15 @@ pub fn file_sha256_digest_string(path: impl AsRef<Path>) -> std::io::Result<Stri
     Ok(digest_string)
 }
 
+/// Return the hex-encoded SHA256 digest of the given bytes.
+pub fn sha256_digest_string(data: &[u8]) -> String {
+    let mut sha = sha2::Sha256::new();
+    sha.update(data);
+    let digest_value = sha.finalize();
+    let digest_string = format!("{:x}", digest_value);
+    digest_string
```

Comment:
This doesn't seem any less clear to me:

```suggestion
    format!("{:x}", Sha256::digest(data))
```

Reply:
TIL (or maybe I already L-ed it but forgot) - thanks!